src/nanoemoji/svg.py

```diff
@@ -315,8 +319,14 @@ def _add_glyph(svg: SVG, color_glyph: ColorGlyph, reuse_cache: ReuseCache):
             reuse_cache.shapes[reuse_key] = el
             for reuse in painted_layer.reuses:
                 _ensure_has_id(el)
-                svg_use = etree.SubElement(svg_g, "use")
-                svg_use.attrib["href"] = f'#{el.attrib["id"]}'
+
+                # The xlink:href attribute must be added to the <use> element, 
+                # otherwise it will not take effect in Safari/Mac/iOS.
```

Comment:
remove the comment, the use of unqualified `href` without the xlink namespace was our own misunderstanding of the spec. Even picosvg always uses `xlink:href` for `<use>` element. It's the way it is, no need to mention Safari, etc